uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -150,4 +163,9 @@ public static void copyApkForPackage(@NonNull final String packageName,
     public static void copyFile(@NonNull final String srcFile, @NonNull final String destFile) {
         executeShellCommand("cp " + srcFile + " " + destFile);
     }
+
+    public static void copyToSdCard(@NonNull final String srcFile, @NonNull final String destFile) {
+        executeShellCommand("mkdir /sdcard/automation");
```

Comment:
You may want to check for existence, or use -p to tolerate it if it's already there.
```suggestion
        executeShellCommand("mkdir -p /sdcard/automation");
```

Reply:
Addressed